doc/basic_configuration.rst

```diff
@@ -17,10 +17,17 @@ configuration to generate the necessary Jobs on the Jenkins master.
 Create / fork configuration repository
 --------------------------------------
 
-First you need to either fork the
+First you need to either fork or clone the
 `ros_buildfarm_config <https://github.com/ros-infrastructure/ros_buildfarm_config>`_
 repository or create a repository containing the same configuration files.
 
+**Important:**
+Since ``ros_buildfarm_config`` files need to be accessed from within docker images,
+you **can not** use a local file system path (`file://`) to reference them.
+You have to provide an http server where the configuration files can be accessed.
+(Note that on the build farm master, Jenkins usually occupies the standard port 80).
```

Comment:
I don't see how the port of Jenkins is relevant here? I don't think you want to start an HTTP server for this.

Reply:
Well, you **have** to have an http server. Whether it's github (if you forked the ros_buildfarm_config repo), another server or localhost.
Since you create the jobs on the buildfarm master, I assume some users may try to use http://localhost, which is however occupied by Jenkins.